config/project.yaml

```diff
@@ -4674,12 +4674,12 @@ sections:
         enabledByDefault: true
         hasUrls: true
         template: ''
-        uriFormat: 'funding/publications/{programme.first.slug}/{slug}'
+        uriFormat: 'funding/publications/{programme.one.slug}/{slug}'
```

Comment:
Ah, _that's_ where it was coming from. Nice 👍

Reply:
It was visible in the stack trace, kind of (you could see it was this section), but buried...